pulsar-io/azure-data-explorer/src/test/resources/sinkConfig.yaml

```diff
@@ -0,0 +1,34 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+{
+  "clusterUrl":"https://somecluster.eastus.kusto.windows.net",
+  "database":"somedb",
+  "table": "tableName",
+  "appId": "xxxx-xxxx-xxxx-xxxx",
+  "appKey": "xxxx-xxxx-xxxx-xxxx",
+  "tenantId": "xxxx-xxxx-xxxx-xxxx",
+  "managedIdentityId": "xxxx-some-id-xxxx OR empty string",
+  "mappingRefName": "mapping ref name",
+  "mappingRefType":"CSV",
+  "flushImmediately":false,
+  "managedIngestion":false,
+  "batchSize":"100",
```

Comment:
why are these numerical strings ? Cant they be plain types like flushImmediately which are boolean types and not "string" booleans